projects/packages/google-fonts-provider/composer.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "automattic/jetpack-google-fonts-provider",
+	"description": "WordPress Webfonts provider for Google Fonts",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.3",
+		"automattic/jetpack-changelogger": "^3.0"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"php -dpcov.directory=. ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
```

Comment:
```suggestion
	"extra": {
		"autotagger": true,
		"mirror-repo": "Automattic/jetpack-google-fonts-provider",
		"textdomain": "jetpack-google-fonts-provider",
		"changelogger": {
			"link-template": "https://github.com/Automattic/jetpack-google-fonts-provider/compare/v${old}...v${new}"
		},
```

Reply:
Done in https://github.com/Automattic/jetpack/pull/23045/commits/21e31d128ab3389d819a7a1314ee9d116f9fee61